virttest/libvirt_xml/pool_capability_xml.py

```diff
@@ -0,0 +1,86 @@
+"""
+Module simplifying manipulation of XML described at
+https://libvirt.org/formatstoragecaps.html
+"""
+
+from virttest.libvirt_xml import base
```

Comment:
it sound better if put it into currently existed ./virttest/libvirt_xml/pool_xml.py